src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py

```diff
@@ -0,0 +1,156 @@
+from base64 import b64encode
+import requests
+from requests.utils import to_native_string
+
+from azure.cli.core.prompting import prompt, prompt_pass, NoTTYException, prompt_y_n
+import azure.cli.core.azlogging as azlogging
+from azure.cli.core.util import CLIError
+
+from ._utils import (
+    get_resource_group_name_by_registry_name,
+    get_registry_by_name
+)
+from .credential import acr_credential_show
+
+logger = azlogging.get_az_logger(__name__)
+APIUSERNAME = ""
+APIPASSWORD = ""
```

Comment:
Can we consolidate the two (and also the API) to use some secret string and take it from an environment variable?